### ISSUE TITLE
Compatibility for current MC version (12.10.2020)

### ISSUE THREAD

**Issue by l-althueser** [XENON1T/processing#69]
PR [230 in the MC repo](https://github.com/XENON1T/mc/pull/230) will break compatibility with the processing scripts. Current workaround is to delete all related options from both python files.
We have to decide how to move forward - we probably need a logic so that 1T simulations are not affected (if we would just drop the options in all files ..). @ramirezdiego what do you think?

**Comment by ramirezdiego** [comment]
I agree with deleting all the electric field options, stating in the README file that one would need a previous version of the processing scripts for the XENON1T full chain (field values are also fetched by the patch scripts from the G4 output file, right? But these haven't been used in a while...).
In addition to that, I was thinking about just moving the processing/montecarlo folder to its own repo in the XENONnT page and do a massive cleanup, maybe constraining it to nT simulations chain. It's probably a lot of copy-pasting, but I was thinking it could help other users understand and maintain these files. You're the expert on this, so feel free to tell me if that's not a good idea.

**Comment by l-althueser** [comment]
See this branch for a workaround: https://github.com/XENON1T/processing/tree/no_efield
